src/components/views/Login.js

```diff
@@ -56,11 +56,11 @@ const Login = () => {
       localStorage.setItem('id', response.data.id)
 
       // Login successfully worked --> navigate to the route /game in the GameRouter
-      history.push(`/game/menu`);
+      history.push(`/game/select/blackCard`);
```

Comment:
ok. later, we have to study when we want to go to the black card and when to go to the menu